pkg/server/chaosd/process.go

```diff
@@ -65,13 +66,21 @@ func (processAttack) Recover(exp core.Experiment, _ Environment) error {
 	}
 	pcmd := config.(*core.ProcessCommand)
 	if pcmd.Signal != int(syscall.SIGSTOP) {
-		return core.ErrNonRecoverableAttack.New("only SIGSTOP process attack is supported to recover")
-	}
+		if pcmd.RecoverCmd == "" {
+			return core.ErrNonRecoverableAttack.New("only SIGSTOP process attack is supported to recover")
```

Comment:
need to update the error message, all process attack support recover

Reply:
Error message has been updated